scripts/install/usr/lib/systemd/system/m0d@.service

```diff
@@ -32,3 +32,5 @@ Environment=SHELL=/bin/bash
 EnvironmentFile=/etc/sysconfig/m0d-%i
 Group=motr
 ExecStart=/usr/libexec/cortx-motr/motr-server m0d-%i
+ExecStop=/bin/bash -c "/usr/bin/echo 'killing PID : ${MAINPID}'"
```

Comment:
Why don't directly exec echo:
```
ExecStop=/usr/bin/echo 'killing PID : ${MAINPID}'
```
?
What is the reason to run it in a bash shell?